serverr/userschema.js

```diff
@@ -0,0 +1,23 @@
+var mongoose = require('mongoose');
+
+var mySchema = new mongoose.Schema({
+    email: String,
+    password: String,
+    askPassword: Boolean,
+    isLog: Boolean
+});
+
+var usersModelColletion = mongoose.model('users', mySchema);
+
+var setUserInformation = (email, hashPassword, askPassword, isLog) => {
+    return new usersModelColletion({
+        email: email,
+        password: hashPassword,
+        askPassword: askPassword,
+        isLog: isLog
+    })
+}
```

Comment:
```suggestion
var setUserInformation = (email, hashPassword, askPassword, isLog) => new usersModelColletion({
        email: email,
        password: hashPassword,
        askPassword: askPassword,
        isLog: isLog
    });
```
immediate return if you're not doing anything else in the function